### DISCUSSION TITLE
Is relative path for `$ref` available?

### DISCUSSION THREAD

**Discussion by ghost** [skryukov/skooma#26] [Q&A] [answered]
I'm facing a problem while using [rspec response matcher](https://github.com/skryukov/skooma?tab=readme-ov-file#validate-request) with OpenAPI schema in which I utilize relative paths for `$ref` like,

```yaml
paths:
  /maintenance_messages:
    $ref: ./paths/maintenance_messages/index.yml
```

With schema like this, the content of `$ref` target file does not seem to be merged into the loaded schema.
I've read document of internally used json_skooma gem, and noticed [`$ref` with relative path is not referred in the README](https://github.com/skryukov/json_skooma/tree/f7326d145e7c6fac7e50b9e150df22a94c7ac1fb?tab=readme-ov-file#handling-external-refs-in-json-schemas).

So, I assume it is impossible to use relative path like above example with this gem. Is this assumption correct?

**Comment by skryukov** [comment, marked as answer]
Hey, @yukihr! Yup, you are right about it, skooma doesn't support external refs atm, but I have this feature on my radar.

**Reply by ghost** [reply]
Thank you for clarification 😄
